client/src/components/Footer/Footer.js

```diff
@@ -1,60 +1,94 @@
+              rel="noopener noreferrer"
+              href="https://github.com/houseofgeeks"
+            >
+              <Github className="icons"></Github>
+            </a>
+            <a
+              className="text-white"
+              target="_blank"
+              rel="noopener noreferrer"
+              href="https://www.facebook.com/hG.iiitranchi"
+            >
+              <Facebook className="icons"></Facebook>
+            </a>
 
-                              <a className="text-white" target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/channel/UCvbh6phGLwpOnoM8g_GwlmQ">
-                                    <Youtube className="icons"></Youtube>
-                              </a>
-                              <a className="text-white" target="_blank" rel="noopener noreferrer" href="https://github.com/houseofgeeks">
-                                    <Github className="icons"></Github>
-                              </a>
-                              <a className="text-white" target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/hG.iiitranchi">
-                                    <Facebook className="icons"></Facebook>
-                              </a>
-
-                              <a className="text-white" target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/company/hgiiitranchi/">
-                                    <LinkedIn className="icons fa-7x"></LinkedIn>
-                              </a>
-                        </div>
-
-                  </div>
-                  <div className="row">
-                        <div className="col-12 col-md-4 col-sm-4 text-center text-white">
-                              <p>WebDev Team House Of Geeks</p>
-                        </div>
-                        <div className="col-12 col-md-4 col-sm-4 text-center text-white">
-                              <p>Made with
-                                    <span role="img" aria-label="heart">
-                                       ❤️️
-                                    </span>
-                                    and open-sourced</p>
-
-                        </div>
-                        <div className="col-12 col-md-4 col-sm-4 text-center text-white">
-                              <p>© HouseOfGeeks,IIIT Ranchi</p>
-                        </div>
-                  </div>
-            </div>
-      </div>);
+            <a
+              className="text-white"
+              target="_blank"
+              rel="noopener noreferrer"
+              href="https://www.linkedin.com/company/hgiiitranchi/"
+            >
+              <LinkedIn className="icons fa-7x"></LinkedIn>
+            </a>
+          </div>
+        </div>
+        <div className="row">
+          <div className="col-12 col-md-4 col-sm-4 text-center text-white">
+            <p data-toggle="modal" data-target="#exampleModal">
+              WebDev Team House Of Geeks

```

Comment:
Lets have some hover effect on this to make its action more obvious to the user.

Reply:
Issue resolved